alternator/server.cc

```diff
@@ -414,7 +414,13 @@ future<> server::init(net::inet_address addr, std::optional<uint16_t> port, std:
             if (https_port) {
                 set_routes(_https_server._routes);
                 _https_server.set_content_length_limit(server::content_length_limit);
-                _https_server.set_tls_credentials(creds->build_server_credentials());
+                _https_server.set_tls_credentials(creds->build_reloadable_server_credentials([](const std::unordered_set<sstring>& files, std::exception_ptr ep) {
+                    if (ep) {
+                        slogger.warn("Exception loading {}: {}", files, ep);
+                    } else {
+                        slogger.info("Reloaded {}", files);
```

Comment:
You can use info0() instead to avoid having 98 repetitions of the message.

Reply:
Yes, but that would generate potential confusion with the warning on exception. What can happen is a user only replaces the PEM and not the KEY file of a cert pair, than you will get an exception "mismatched certs". Then he completes his file mods and replaces KEY as well, so reload succeeds, and we print the joyful message. 
But if this was in a race condition where some shard is lagging, we would get could get a shard0 "success", but a shardX fail (followed by a silence success). Me thinks confusing to log reader...